### PR TITLE
Add swagger and Lombok

### PR DESCRIPTION
**Pull request by Gradder**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,13 +21,16 @@ dependencies {
...
+    annotationProcessor("io.micronaut.openapi:micronaut-openapi:2.1.1")
     implementation(platform("io.micronaut:micronaut-bom:$micronautVersion"))
     implementation("io.micronaut:micronaut-inject")
     implementation("io.micronaut:micronaut-validation")
     implementation("io.micronaut:micronaut-runtime")
     implementation("javax.annotation:javax.annotation-api")
     implementation("io.micronaut:micronaut-http-server-netty")
     implementation("io.micronaut:micronaut-http-client")
+    implementation("io.swagger.core.v3:swagger-annotations")
+    compile group: 'io.micronaut', name: 'micronaut-management', version: '2.0.3'
```
Comment (goponenkoev) [inline build.gradle]: Let's align with the same style as before

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    static-resources:
+      swagger:
+        paths: classpath:META-INF/swagger
+        mapping: /swagger/**
+      swagger-ui:
+        paths: classpath:META-INF/swagger/views/swagger-ui
+        mapping: /swagger-ui/**
+
+endpoints.all.sensitive: false
+
```
Comment (goponenkoev) [inline src/main/resources/application.yml]: Is it really need to have such many empty lines below? :)

**Review by goponenkoev — COMMENTED** [review]
Could you please also provide some small explanation in Readme how to run swagger and see available endpoints?

File: build.gradle (hunk 1 above)
Comment (Gradder) [reply build.gradle]: Done

File: src/main/resources/application.yml (hunk 2 above)
Comment (Gradder) [reply src/main/resources/application.yml]: Of course!

**Review by goponenkoev — APPROVED** [review]
Looks good to me

**Pull request merged**